autoswitch_virtualenv.plugin.zsh

```diff
@@ -277,6 +277,12 @@ function _missing_error_message() {
     printf "\n"
 }
 
+function randstr()
+{
+    local strlen="${1:=4}"
+    python -c "from __future__ import print_function; import string, random; print(''.join(random.choice(string.ascii_lowercase) for _ in range($strlen)))"
```

Comment:
I actually can't think of a situation where we ever don't want this to be fixed to 4. So in order to prevent code injection attacks could you just hardcode that value instead of taking it as a parameter?